mockserver-core/src/main/java/org/mockserver/echo/http/EchoServerHandler.java

```diff
@@ -106,7 +106,7 @@ public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
         mockServerLogger.logEvent(
             new LogEntry()
                 .setLogLevel(Level.ERROR)
-                .setMessageFormat("echo server server caught exception")
+                .setMessageFormat("echo server caught exception")
```

Comment:
It's possible that `server server` is intentional, but it's hard to see why and hopefully it could be changed to something else. E.g.` echo-server server caught`.

Here, there's an _echo server_, while it _is_ a server, I don't think that needs to be repeated.